src/applications/find-forms/helpers/selectors.js

```diff
@@ -0,0 +1,8 @@
+// import the toggleValues helper
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
+
+export const getFormState = state => state.findVAFormsReducer;
```

Comment:
Good idea to make a selector! Small suggestion on the name - we work with a lot of forms on the site so "form state" typically refers to the values of fields in a form. I think I would do the more verbose `getFindFormsAppState` or something like that